docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -285,10 +285,18 @@ role ``meta/argument_specs.yml`` file. All fields are lower-case.
 
         * A short, one-line description of the entry point.
         * The ``short_description`` is displayed by ``ansible-doc -t role -l``.
+        * The short description should always be a string and never a list.
 
     :description:
 
         * A longer description that may contain multiple lines.
+        * Every new list item is a new paragraph. You can have multiple sentences per paragraph.
```

Comment:
```suggestion
        * This can be a single string, or a list of strings. In case this is a list of strings, every list
           element is a new paragraph.
```
I would probably skip `You can have multiple sentences per paragraph.` here; I only added it in the example to emphasize that you don't have to put every sentence in one paragraph (because without this sentence the example did exactly that :) ).